pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -307,37 +307,49 @@ func ResourceConfigFromReply(reply *lifecycle.EventReply, resources *ResourceCon
 
 // Updates the supplied container config in-place based on the isolation
 // controls in the event reply.
-func UpdateContainerConfigWithReply(reply *lifecycle.EventReply, config *runtime.ContainerConfig) {
+func UpdateContainerConfigWithReply(reply *lifecycle.EventReply, config *runtime.ContainerConfig) *runtime.ContainerConfig {
+	if reply == nil {
+		glog.Info("eventDispatcherNoop has been detected - skipping container configuration update")
+		return config
+	}
+	if reply.Error != "" {
+		glog.Errorf("isolator returned error: %s", reply.Error)
+		return config
```

Comment:
We should return an error in this case and let the caller decide how to handle it. Probably, the caller (kuberuntime) needs to throw the error off the top in order to interrrupt container launch. We do not want to swallow this error here.